host/xtest/Makefile

```diff
@@ -77,6 +77,14 @@ ifeq ($(CFG_SECURE_DATA_PATH),y)
 srcs += sdp_basic.c
 endif
 
+ifeq ($(CFG_SECURE_KEY_SERVICES),y)
+srcs += regression_4200.c
+OPTEE_SKS_HEADERS ?= ../../../optee_client/libsks/include
```

Comment:
Please don't reach outside of this git into some other git for internal .h files. Whatever .h files are needed should be installed somewhere where we can find them.

Reply:
ok. I will remove this line and `CFLAGS += -I$(OPTEE_SKS_HEADERS)` below.
The lib headers are already exported in `$(OPTEE_CLIENT_EXPORT)/include`.